### ISSUE TITLE
Duplicate ids causing accessibility warnings

### ISSUE THREAD

**Issue by thanyabegum** [projectblacklight/spotlight#2788]
There are duplicate ids in the tabs of the Configuration page of an exhibit and in the bulk actions of the search results page that have failed our accessibility checks (https://github.com/pulibrary/dpul/issues/1227 and https://github.com/pulibrary/dpul/issues/1264).
